crypto/x509/v3_utl.c

```diff
@@ -847,8 +850,13 @@ static int do_check_string(const ASN1_STRING *a, int cmp_type, equal_fn equal,
             return -1;
         }
         rv = equal(astr, astrlen, (unsigned char *)b, blen, flags);
-        if (rv > 0 && peername)
+        if (rv > 0 && peername) {
```

Comment:
While you are modifying this, could you please change this to `peername != NULL` to match the coding style.

Reply:
Fine, I have submitted a fixup to change them.